spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaListenerAnnotationBeanPostProcessor.java

```diff
@@ -509,11 +510,15 @@ private boolean processMainAndRetryListeners(KafkaListener kafkaListener, Object
 		return true;
 	}
 
+	@SuppressWarnings("deprecation")
```

Comment:
Are you sure that we really have something in this method to suppress?

Reply:
I'll double check, but I think we had some bean names. Mind that I deprecated the entire `RetryTopicInternalBeanNames` class, so some suppressed deprecations are because of that.

Reply:
Yep, bean names it is.